templates/config.tfvars.azure-template

```diff
@@ -19,7 +19,7 @@ node_vm_size = "Standard_DS2_v2"
 # Gluster configuration
 glusternode_count = "1"
 glusternode_vm_size = "Standard_DS2_v2"
-glusternode_extra_disk_size = "200" # Size in GB
+glusternode_extra_disk_size = "250" # Size in GB
```

Comment:
Increased `glusternode_extra_disk_size` to 250 GB in order to allow all PVC to be allocated correctly when all services are enabled at once.